### ISSUE TITLE
createFeatureSelector<T, V> not support in ngRX Store 6.01

### ISSUE THREAD

**Issue by mattiLeBlanc** [ngrx/platform#1229]
Hi,

I was looking at your example app. I am using NgRX6 myself and I noticed that you provide 2 interfaces in the `app`\reducers\index.ts` when defining the `getLayoutState` feature selector.
```
export const getLayoutState = createFeatureSelector<State, fromLayout.State>(
  'layout'
);
```
This feature is not yet available in ngrx 6.01. Is this going to be corrected soon? It might be related to an issue I have creating a feature selector on my rootStore, which is not returning the State object but the Feature Object. I doing the same thing as in the example app, other than that the createFeatureSelector function is a tad different.

**Comment by brandonroberts** [comment]
We released `6.1` last week that includes this feature.

**Comment by mattiLeBlanc** [comment]
Wow you guys are quick. I must have installed my version just a couple of days before :) :).
Great stuff!
